src/b-st-ext.adoc

```diff
@@ -5,12 +5,11 @@ The B standard extension comprises instructions provided by the Zba, Zbb, and
 Zbs extensions.
 
 [[preface]]
-=== Bit-manipulation a, b, c and s extensions grouped for public review and ratification
+=== Bit-manipulation small extensions
```

Comment:
"small extensions" isn't a term/concept we should proliferate.  I suggest changing the title to " Bit-manipulation Extensions Zba, Zbb, Zbb, and Zbs"

Reply:
I wonder if it makes sense to include a title for this part at all since it’s really an intro to all the Zb* extensions (including Zbk*). Maybe just having it under the main chapter title is fine. 

If we do want a title, what about “Zb* Extensions Overview” or something along those lines?

Reply:
If you think ditching the title altogether works, that’s fine with me, too.

Reply:
Looking at it more closely, the Zc* chapter uses "Zc* Overview" to title a similar section, so I went with that. If you'd rather I removed it let me know and I can make that change.